stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt

```diff
@@ -692,12 +692,12 @@ public class MessageComposerController(
         /**
          * The regex pattern used to check if the message ends with incomplete mention.
          */
-        private val MentionPattern = Pattern.compile("^(.* )?@([a-zA-Z]+[0-9]*)*$", Pattern.MULTILINE)
+        private fun mentionPattern() = Pattern.compile("^(.* )?@([a-zA-Z]+[0-9]*)*$", Pattern.MULTILINE)
```

Comment:
Hey @leandroBorgesFerreira ,

I'm a bit concerned about this change.

Based on Pattern docs:
```
repeated matches it is less efficient since it does not allow the compiled pattern to be reused.
```

I think if that would be a problem Google didn't add static compiled Patterns at `android.util.Patterns`.

Reply:
You're right, this can happen very often. What about moving this to the class instead of the `companion object`?

Reply:
Moved to class variable